bin/reth-bench/README.md

```diff
@@ -0,0 +1,63 @@
+# Benchmarking reth live sync with `reth-bench`
+
+The binary contained in this directory, `reth-bench`, is a tool that can be used to benchmark the performance of the reth live sync. `reth-bench` is a general tool, and can be used for benchmarking node performance, as long as the node supports the engine API.
```

Comment:
```suggestion
`reth-bench`, is a tool designed to benchmark performance of reth live sync. It can also be used for spec debugging, as it allows replaying blocks by mocking CL. Performance is measured by latency and gas required to execute a block, as well as gigagas <which we care about as a measure of perf because...>
```

also missed adding this to the review @Rjected, please fill out `<which we care about as a measure of perf because...>`.

background was missing on (1) how perf is measured (units), and (2) why do we care ab this? audience being people who are knowledgeable about the task, but appreciate the small memory jogger before digging in

Reply:
sg, added some description here